galaxy_ng/app/api/v3/urls.py

```diff
@@ -81,6 +81,15 @@
         name="collection-artifact-upload",
     ),
 
+    # This endpoint is defined here so we can temporarily override the serializer
+    # to avoid displaying signatures due to AAH-1932
+    path(
```

Comment:
Do you need to override the list serializer for collection versions too?

Reply:
The list collection version list serializer does not include signatures, so i left that one alone